main.go

```diff
@@ -81,7 +92,60 @@ func main() {
 	fmt.Println()
 	log.Infof("$ apt-get %s", command.PrintableCommandArgs(false, cmdArgs))
 	if err := command.RunCommand("apt-get", cmdArgs...); err != nil {
-		log.Errorf("Can't install packages:  %s", err)
-		os.Exit(1)
+		fail("Can't install packages:  %s", err)
+	}
+}
+
+func applyCacheConfig(cacheConfig string) error {
+	switch cacheConfig {
+	case cacheInputNone:
+		return applyNoCache()
+	case cacheInputAll:
+		return applyAllCache()
+	default:
+		return fmt.Errorf("invalid cache level, no such configuration: %s", cacheConfig)
+	}
+}
+
+func applyNoCache() error {
```

Comment:
please do not do this. because the user don't want to upload their package cache to the cache archive doesn't mean they don't want to maybe reuse them in the current build session. especially when steps are fetching their apt-get packages, this would make the builds faster

Reply:
Also, none != disable